case-study-1-food-delivery-example-1/services/orders/orders/app.py

```diff
@@ -33,56 +32,50 @@ def foo_health_check():
     return jsonify({ "status": "OK" })
 
 
-@app.route("/orders/create", methods=['GET'])
-def create():
-
-    #declare a random order (in practice, these details would be passed into the function)
-    auth_type = "create"
-    order_details = {"id" : 1234, "amount" : 9.99}
+@app.route("/orders", methods=['POST'])
+def create_order():
 
-    return get_auth(auth_type, order_details) 
+    #get the order amount from the request and assign the order id
+    data = request.json      
+    order_amount = data['order_amount']  
+    order_details = {"order_id": 1234, "order_amount": order_amount}
```

Comment:
Doesn't this mean no matter how many times we run the test every single order id is going to be the same?  Perhaps keep a counter in the app and increment it each time so that it remains deterministic, but the order ids are different.